portability-types-common/src/main/java/org/datatransferproject/types/common/models/DateRangeContainerResource.java

```diff
@@ -0,0 +1,57 @@
+/*
+ * Copyright 2019 The Data Transfer Project Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.datatransferproject.types.common.models;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonTypeName;
+
+import java.util.Objects;
+
+/** A date range container containing start and end date for filtering. */
+@JsonTypeName("org.dataportability:DateRangeContainerResource")
+public class DateRangeContainerResource extends ContainerResource {
+  private final Integer startDate;
+  private final Integer endDate;
+
+  @JsonCreator
+  public DateRangeContainerResource(
+          @JsonProperty("startDate") Integer startDate,
```

Comment:
We thought since we are looking to pass Unix timestamps for dates, it might be best to pass them as integers timestamps. 

We did consider using String for this purpose as well but thought it may not be as obvious as to the format in which the date is being passed.